.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 4, 0, 2, "final", 0
+current_version = 4, 1, 0, "alpha", 0
```

Comment:
Will that not cause an issue when releasing the first `4.1.0a0` with the releaser?

Reply:
Or we need to start at `4.1.0a1`